crates/bevy_pbr/src/extended_material.rs

```diff
@@ -0,0 +1,201 @@
+use bevy_asset::Asset;
+use bevy_reflect::TypePath;
+use bevy_render::{
+    mesh::MeshVertexBufferLayout,
+    render_asset::RenderAssets,
+    render_resource::{
+        AsBindGroup, AsBindGroupError, BindGroupLayout, RenderPipelineDescriptor, ShaderRef,
+        SpecializedMeshPipelineError, UnpreparedBindGroup,
+    },
+    renderer::RenderDevice,
+    texture::{FallbackImage, Image},
+};
+
+use crate::{Material, MaterialPipeline, MaterialPipelineKey, StandardMaterial};
+
+/// A subset of the `Material` trait for use when extending `StandardMaterial`s.
```

Comment:
"for use when" awkward warning, idk what would be better though.

Reply:
"for defining extensions to" ?

Reply:
yeah that works